frontend/package.json

```diff
@@ -9,12 +9,15 @@
     "@types/jest": "^24.9.1",
     "@types/node": "^12.12.37",
     "@types/react": "^16.9.34",
+    "@types/react-big-calendar": "^0.24.0",
     "@types/react-calendar": "^3.0.1",
     "@types/react-dom": "^16.9.6",
     "@types/react-leaflet": "^2.5.1",
     "date-fns": "^2.12.0",
     "leaflet": "^1.6.0",
+    "moment": "^2.25.3",
```

Comment:
I've already started using `date-fns`, since it's more compact than `moment` and supports immutability, while `moment` does not. From what I see, `react-big-calendar` also supports `date-fns`, so I'd rather stick to that.